src/enc-base64url.js

```diff
@@ -24,11 +24,11 @@
          *
          *     var base64String = CryptoJS.enc.Base64url.stringify(wordArray);
          */
-        stringify: function (wordArray, urlSafe=true) {
+        stringify: function (wordArray, urlSafe) {
             // Shortcuts
             var words = wordArray.words;
             var sigBytes = wordArray.sigBytes;
-            var map = urlSafe ? this._safe_map : this._map;
+            var map = urlSafe === undefined ? this._map : this._safe_map;
```

Comment:
I think you want
```
var map = urlSafe === undefined || urlSafe ? this._safe_map : this._map;
```
If `urlSafe` is `undefined`, you want this to default to `true`, so you want to use the `_safe_map` scenario. Or perhaps I'm missing something?

Reply:
If you use babel, you will find that the default value of the function parameter only needs to judge undefined.
![image](https://user-images.githubusercontent.com/32794413/129825670-9e8ab287-7d68-44af-8536-69270617aaaf.png)

Reply:
emmm...you are right, but i think maybe this way will be better?
```ts
var map = urlSafe ? this._safe_map : this._map;
````